tsl/test/expected/continuous_aggs-12.out

```diff
@@ -1964,6 +1964,33 @@ SELECT * FROM mat_m1 ORDER BY 1, 2;
  100 |   100
 (5 rows)
 
+-- Parallel planning for realtime continuous aggregate
+SET force_parallel_mode = 'on';
+SET parallel_setup_cost = 0;
+SET max_parallel_workers_per_gather = 1;
+EXPLAIN (COSTS OFF, TIMING OFF) SELECT * FROM mat_m1;
+                                                                                      QUERY PLAN                                                                                      
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Shouldn't we see "workers planned" here somewhere? Meaning that the parallel plan is chosen.

Reply:
The text "Workers Planned" is only printed if a Gather or GatherMerge node is added, and since it is set to max 1, that probably explains it. Might be better to set it to something higher.

Reply:
Reworked the tests!